static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -127,6 +133,12 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
      * @type {Object}
      */
     this.alternativeVerticalsConfig = config.alternativeVerticalsConfig;
+
+    /**
+     * Whether the page is displaying the map on mobile.
+     * @type {boolean}
+     */
+    this.isInMobileMapView = false;
```

Comment:
This is going to have a conflict with #703 
We may want to update to use the MapStates enum

Reply:
oh ok I'll wait for 703 to be merged in

Reply:
then rebase

Reply:
703 has been merged, so you should be good to rebase

Reply:
just realized 703 is an area code